spec/models/activity_spec.rb

```diff
@@ -416,6 +416,16 @@
       end
     end
 
+    context "when planned_end_date is not blank" do
+      let(:activity) { build(:activity) }
+
+      it "does not allow planned_end_date to be earlier than planned_start_date" do
+        activity = build(:activity, planned_start_date: Date.today, planned_end_date: 1.day.ago)
```

Comment:
For consistency with your other specs which use `Date.today` and `Date.tomorrow`, you could use `Date.yesterday` here.

Reply:
That's true! I didn't notice. Thanks for that!